content/en/docs/api_reference/package-operator-api.md

```
@@ -897,6 +972,135 @@ Used in:
 * [ObjectTemplateStatus](#objecttemplatestatus)
 
 
+### HostedClusterPackagePartitionOrderSpec
+
+HostedClusterPackagePartitionOrderSpec describes ordering for a partition.
+
+| Field | Description |
+| ----- | ----------- |
+| `static` <br>[]string | Allows to define a static partition order.<br>The special * key matches anything not explicitly part of the list.<br>Unknown risk-groups or HostedClusters without label<br>will be put into an implicit "unknown" group and<br>will get upgraded LAST. |
+| `alphanumericAsc` <br><a href="#hostedclusterpackagepartitionorderalphanumericasc">HostedClusterPackagePartitionOrderAlphanumericAsc</a> |  |
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe has extra space to the right for style "compact"

(MD060, table-column-style)